### ISSUE TITLE
Pages: add button is read wrongly as "Button. Possibly add." with VoiceOver

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12870]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to My Site > Site Pages.
1. Swipe right until you reach the add button and observe how it is read out.

**Result:** the add button is read wrongly as "Button. Possibly add." and should be something like. "Add page. Button." ([2s](https://cloudup.com/cM_A3uh0zmw))

<img src="https://user-images.githubusercontent.com/1119271/68251836-309f9500-ffe1-11e9-97b3-aa4029f585e8.png" width="270" alt="pages-add-button">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by shiki** [comment]
Fixed by @vphong in #13057.
